autoload/firenvim.vim

```diff
@@ -185,13 +185,12 @@ function! s:key_to_ps1_str(key, manifest_path)
         let l:key_arr = split(a:key, '\')
         let l:i = 0
         for l:i in range(2, len(l:key_arr) - 1)
-                let l:ps1_content = l:ps1_content . "\nNew-Item -Path \"" . join(key_arr[0:i], '\') . '" -ErrorAction SilentlyContinue'
+                let l:ps1_content = l:ps1_content . "\nNew-Item -Path \"" . join(key_arr[0:i], '\') . '"'
         endfor
         " Then, assign a value to it
         return l:ps1_content . "\nSet-Item -Path \"" .
                                 \ a:key .
-                                \ '\" -Value "' . a:manifest_path . '" ' .
-                                \ '-ErrorAction SilentlyContinue'
```

Comment:
I remember `-ErrorAction SilentlyContinue` being necessary in order to be able to create keys if part of the path already existed. I guess I must have done something wrong when I worked on this.

Reply:
I saw such errors/warnings when running the script/commands directly, but they don't seem to short-circuit the later commands.